README.md

```diff
@@ -73,8 +150,83 @@ NAME                                        READY   AGE
 statefulset.apps/ibm-block-csi-controller   1/1     9m36s
 ```
 
+
+## Configuring k8s secret and storage class 
+In order to use the driver, create the relevant storage classes and secrets, as needed.
+
+This section describes how to:
+ 1. Create a storage system secret - to define the storage credential (user and password) and its address.
+ 2. Configure the k8s storage class - to define the storage system pool name, secret reference, SpaceEfficiency (thin, compressed, or deduplicated) and fstype(xfs\ext4).
+
+#### 1. Create an array secret 
+Create a secret file as follows and update the relevant credentials:
+
+```
+kind: Secret
+apiVersion: v1
+metadata:
+  name: <VALUE-1>
+  namespace: kube-system
+type: Opaque
+stringData:
+  management_address: <VALUE-2,VALUE-3> # Array management addresses
+  username: <VALUE-4>                   # Array username
+data:
+  password: <VALUE-5 base64>            # Array password
+```
+
+Apply the secret:
+
+```
+$ kubectl apply -f array-secret.yaml
+```
+
+#### 2. Create storage classes
+
+Create a storage class yaml file `storageclass-gold.yaml` as follows, with the relevant capabilities, pool and, array secret:
+
+```bash
```

Comment:
it is not a script, so I think you don't need to add "bash" here.

Reply:
fixed